### ISSUE TITLE
Standalone always crash

### ISSUE THREAD

**Issue by fuadxtps23** [jameshball/osci-render#256]
Just messing around with blender and osci-render but when my focus is to blender, the osci-render always crashes

**Comment by loglot** [comment]
yeah, this has happened to me when focusing on any app that isn't osci-render, however, it's not consistent, and it doesn't really happen with blender
